src/main/resources/fabric.mod.json

```diff
@@ -12,7 +12,7 @@
   },
   "accessWidener": "spleef.accesswidener",
   "depends": {
-    "fabricloader": ">=0.9.1",
+    "fabricloader": ">=0.12.12",
```

Comment:
Why is `0.12.12` being depended on here?

Reply:
This was done as `0.9.1` has been out for a while. I believe you need `0.12.0` to run a 1.18 server anyways. I chose this version as it includes patches for all log4shell vulnerabilities but the requirement can be lowered or dropped and it should be enforced by Plasmid anyways.